h2o-py/h2o/model/metrics_base.py

```diff
@@ -128,11 +128,11 @@ def show(self):
             print("Mean Residual Deviance: " + str(self.mean_residual_deviance()))
         if metric_type in types_w_logloss:
             print("LogLoss: " + str(self.logloss()))
-        if metric_type in ['ModelMetricsBinomial', 'ModelMetricsBinomialGeneric']:
+        if metric_type in ['ModelMetricsBinomial', 'ModelMetricsBinomialGeneric'] or \
+            metric_type in types_w_mult or \
+            metric_type in ['ModelMetricsOrdinal', 'ModelMetricsOrdinalGeneric']:
```

Comment:
recommended way for multiline statements in Py is to wrap everything with parenthesis instead of escaping end-lines. Also, starting next line with `or` makes it more clear that it's a continuation of the previous line imo.

Reply:
thank you - fixed